ios/FluentUI.Demo/FluentUI.Demo/Demos/SearchBarDemoController.swift

```diff
@@ -13,11 +13,18 @@ class SearchBarDemoController: DemoController, SearchBarDelegate {
     }()
 
     private lazy var badgeView: UIView = {
-        let dataSource = BadgeViewDataSource(text: "Kat Larsson")
+        let imageView = UIImageView(image: UIImage(named: "avatar_kat_larsson"))
```

Comment:
we should have both cases in the test app which has avatar image in the badgeview and not have avatar in the bage